node/src/actors/chain_manager/actor.rs

```diff
@@ -177,4 +175,22 @@ impl ChainManager {
             })
             .wait(ctx);
     }
+
+    /// Load public key hash from signature manager
+    fn get_pkh(&mut self, ctx: &mut Context<Self>) {
+        signature_mngr::pkh()
+            .into_actor(self)
```

Comment:
*Suggestion*: `map_error` could be moved on top of `into_actor` and you avoid declaring the `_act, _ctx` params